### PR TITLE
[HOPSWORKS-1001] Experiments service refactor certificates cleanup

### PR DESCRIPTION
**Pull request by robzor92**

### CODE REVIEW

**Comment by robzor92** [comment]
https://github.com/logicalclocks/hops-testing/pull/348

**Comment by robzor92** [comment]
@tkakantousis please review.

File: templates/default/tensorboard-cleanup.sh.erb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  help
+fi
+
+if [ ! -d "$TENSORBOARD_DIR" ] ; then
+    echo "Secret dir does not exist: $TENSORBOARD_DIR"
+    exit 2
+fi
+
+# Remove the TensorBoard directory in the staging directory
+if [ $TENSORBOARD_DIR != "" ] ; then
```
Comment (tkakantousis) [inline templates/default/tensorboard-cleanup.sh.erb]: @robzor92 this could be one clause with an `AND` operator.

Comment (robzor92) [reply templates/default/tensorboard-cleanup.sh.erb]: Fixed

**Pull request merged**